### ISSUE TITLE
New-HPOVUplinkSet for a FCoE network on a LIG fails

### ISSUE THREAD

**Issue by VincentBerger** [HewlettPackard/POSH-HPOneView#473]
### Expected Behavior
```
    $LIGFlex = Get-HPOVLogicalInterconnectGroup -Name "LIG-VCEth"
    $SAN_A_FCoE = Get-HPOVNetwork -Name "SAN A FCoE"
    New-HPOVUplinkSet -Resource $LIGFlex -Name "US-SAN-A-FCoE" -Type Ethernet -Networks $SAN_A_FCoE -UplinkPorts "Enclosure1:Bay3:Q1.1" -LacpTimer Short | Wait-HPOVTaskComplete
```
should work
### Actual Behavior
but it fails. This used to work in previous versions of the library.
```PowerShell
VERBOSE: [GETNETWORKURIS] Called from: BuildUplinkSet
VERBOSE: [GETNETWORKURIS] _net Type is System.Management.Automation.PSCustomObject
VERBOSE: [GETNETWORKURIS] Network 'SAN A FCoE' is [System.Management.Automation.PSCustomObject]
VERBOSE: [GETNETWORKURIS] Network URIs: /rest/fcoe-networks/29000a60-b844-420c-a39d-5dac3fd39ac6
VERBOSE: [GETNETWORKURIS] Done.
New-HPOVUplinkSet: You cannot call a method on a null-valued expression.
````

### Steps to reproduce
From what I could gather from a debugging session, the line that causes the exception is:
```    [void]$_UplinkSetObject.fcoeNetworkUris.Add($_NetworkUri)```
$_UplinkSetObject is either a liUplinkSetObject or a ligUplinkSetObject
you've added the fcoeNetworkUris property to liUplinkSetObject but not to ligUplinkSetObject so when working with a LIG $_UplinkSetObject.fcoeNetworkUris is null and calling the Add method on it throws the exception

### Version Information
HPE OneView PowerShell Library Version : 5.0.2400.1845

**Comment by ChrisLynchHPE** [comment]
Hmm...  Thought I had addressed that, as LIG uplink set properties have a different structure than provisioned uplink sets do not have network specific properties.  They are all added to a single `networkUris` property.  I'll look into this further.

**Comment by VincentBerger** [comment]
You added that logic for FC networks
if working on LIG {add network to networkUris property}
else {add network to fcNetworkUris property}
but that 'if' is missing if the network is FCoE

**Comment by ChrisLynchHPE** [comment]
Yes, I did see that.  It was supposed to be for both.  And unfortunately, in my Unit tests, I don't cover this.  I'll ensure it is a unit test for future releases.

**Comment by SahilMutha** [comment]
I also observed the same issue in the latest library. Any update on when this will be available @ChrisLynchHPE ?

**Comment by ChrisLynchHPE** [comment]
This was addressed, but I forgot to document it, that this is fixed in the [5.20.2452.2750 release](https://www.powershellgallery.com/packages/HPOneView.520/5.20.2452.2750).

**Comment by VincentBerger** [comment]
Fix verified in 5.20.2452. Closing
